### ISSUE TITLE
Imported images do not appear in collections module

### ISSUE THREAD

**Issue by cmjns** [darktable-org/darktable#15645]
### Describe the bug

When copying and importing, the resulting import is nowhere to be found. I've set up naming rules for copying images out of a "Capture" folder on my hard drive to a folder on my external drive. When I choose to copy and import, the photos are copied and renamed correctly but nothing shows in the Collections module at all. I cannot find the imported images using any of the available collection filters. I tried film role, and all the iterations of date filters. It looks for all the world like they just aren't in the DT database. 

Further, I tried manually copying the renamed files to a new directory, from which I tried to bypass the "Copy and Import" option to just "Add to Library". But DT doesn't see the images in that directory. The location selection modal sees the directory on disk but it pretends the directory is empty. 

Is there a way to just list the entries in the database so I can see if they're in there? What would happen if I just hosed those directories that I already imported? 

Is there a way to "reverse" an import or remove files from the library? 

### Steps to reproduce

1. Click Copy and Import
2. Select origin folder on disk
3. Set up a name for the import 
4. Watch files get copied and renamed
5. Look for files in film roll 
6. Files aren't there.
7. Go to "Add to Library"
8. Navigate to folder with copied / imported images
9. Selection modal shows an empty directory even if the option to show only unimported files is unchecked
10. Manually copy copied and imported files to a new directory 
11. Click "Add to Library" 
12. Navigate to the new folder with copied images
13. Modal still shows an empty directory with no importable images

### Where did you install darktable from?

darktable.org

### darktable version

4.4.2

### What OS are you using?

Linux

### What is the version of your OS?

Mint (latest)

### Are you using OpenCL GPU in darktable?

None

**Comment by ralfbrown** [comment]
What is your renaming pattern?  *Does it include the file extension?*

Check the destination folder with a file manager (not dt).  Are your images present and do they have extensions like .jpg, .nef, .cr2, etc.?  darktable's import dialog won't find files that don't have a known file extension for an image type.

**Comment by gi-man** [comment]
> * Look for files in film roll
> * Files aren't there.

Do you have any filters active?

**Comment by ptilopteri** [comment]
does <user> have read/write access to subject directory?

**Comment by cmjns** [comment]
I determined that I'd left off the file extension. I wasted a lot of time on it, too. It would have been extremely helpful to have an error message or some kind of warning to let me know I was doing something that would break things downstream. I'm coming from OSX / Windows where filename extensions are largely hidden and managed by the software, so I overlooked this detail in the naming pattern. If a warning had popped up to say "You left off the extension in your pattern and as a consequence imported images will not be shown in the Collections module" I wouldn't be here days later typing into a github issue queue and not editing my photos. 

I know this isn't the place for a rant but trying to use DT so far has been a nightmare. I keep getting tripped up on minor but extremely consequential usability issues at every turn, that cost me hours or even days. I'm one problem away from going back to windows and LR.
